src/hssm/param.py

```diff
@@ -419,76 +417,43 @@ def make_bounded_prior(prior: ParamSpec, bounds: BoundsSpec) -> float | bmb.Prio
 
     Returns
     -------
-    float | bmb.Prior
-        A float if `prior` is a float, otherwise a bmb.Prior object.
+    float | Prior
+        A float if `prior` is a float, otherwise a hssm.Prior object.
     """
     lower, upper = bounds
 
     if isinstance(prior, float):
```

Comment:
might make sense to outsource all these type-checks into a separate function for readability? Something like _prior_type_checks()

Reply:
ah well looking at it, this function is not doing much beyond that anyways so makes no sense :)